app/views/admin/events/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<% provide(:custom_css, 'event-components/single-page') %>
+
+<%= render 'events/show-partials/intro' %>
+<%= render 'events/show-partials/info' %>
+<%= render 'events/show-partials/partners' %>
+<%= render 'events/show-partials/gallery' %>
```

Comment:
I think that we don't need to put all of those CSS for the admin/events/show, because it will change the database and the display will be the same as in the /events/show. Just put minimum plain design first, I will help to decorate it later using another PR.

Reply:
Okay